### PR TITLE
Forgot adding test to Makefile

### PR DESCRIPTION
**Pull request by ferdymercury**

### CODE REVIEW

File: root/treeformula/scan/Makefile [hunk 1]
```diff
@@ -62,6 +62,9 @@ endif
 # By default the TestDiff variable is set to compare the .log and .ref files
 # To run a diff and ignore whitespacing use $(TestDiffW)
 
+LoadDim: LoadDim.log
```
Comment (pcanal) [inline root/treeformula/scan/Makefile:65]: The change in the `Makefile` can also be replaced by 'just' renaming the script to have an `exec` prefix: `execLoadDim.C` ;  The function should (of course) be also renamed and be made to `return 0` upon success for good measure.  The reference file should also be renamed be `execLoadDim.ref`.

Comment (ferdymercury) [reply root/treeformula/scan/Makefile:65]: Fixed

**Comment by pcanal** [comment]
@phsft-bot build

**Comment by ferdymercury** [comment]
@pcanal I guess failures are unrelated. Thanks for the review!

**Review by pcanal — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**